builder/Makefile.release

```diff
@@ -186,3 +187,20 @@ endif
 clean:
 #
 	$(MAKE) clean
+
+TAGS := latest $(IMAGE_TAG) full full-$(IMAGE_TAG)
+
+define sign_rule
+.PHONY: $1-sign
+$1-sign:
+	COSIGN_EXPERIMENTAL=1 $(CMD_COSIGN) sign $(PUSH_DOCKER_REPO)@$(shell crane digest $(PUSH_DOCKER_REPO):$1)
```

Comment:
can we sign the tag instead of the digest? this way we don't need crane dependency and it's simpler

Reply:
We should sign the digest, not the tag itself.
> https://github.com/sigstore/cosign#sign-a-container-and-store-the-signature-in-the-registry

Reply:
I understand the recommendation, but not sure it applies in our case, since we are using the tag to find the digest anyway. If there's ambiguity in the tag, we will sign the wrong image in both cases (please correct me if I'm wrong). So in this case I think we can just sign the tag in the first place

Reply:
A container image tag is a human-readable string that is associated with a specific version of the image. It is typically in the format of "repository:tag" and is used to identify the image in a registry. For example, an image could have a tag of "nginx:1.19.2" to indicate that it is version 1.19.2 of the nginx image.

A container image digest, on the other hand, is a unique and immutable identifier generated from the image's content. It is a cryptographic hash of the image's metadata and layers. The digest is used to verify the integrity of the image and to ensure that it has not been tampered with. For example, an image could have a digest of "sha256:abcdef123456789".

In general, using the image digest instead of the image tag when pulling images from a registry is recommended, the same applies to signing images, as it provides a stronger guarantee of the image's content. The tag can change for a particular image, for example, when a new version is pushed to the registry, whereas the digest remains the same for a specific version of the image.

> _https://cloud.google.com/kubernetes-engine/docs/archive/using-container-images_

Reply:
I also think we should use image digest and not tag.
@itaysk is your only concern is the crane dependancy?
Can't we then simply use docker inspect to get the image digest instead of crane?

Reply:
This would work yeah, and I updated the code by removing the crane and using the docker inspect

Reply:
I just didn't see the value of signing digest of tag in this specific case (i understand the value generally). anyway, it shouldn't block the pr

Reply:
we are ready for merge then 🙉